pkg/events/events.go

```diff
@@ -64,3 +64,13 @@ func NodeFailedToDrain(node *v1.Node, err error) Event {
 		DedupeValues:   []string{node.Name},
 	}
 }
+
+func NodeInflightCheck(node *v1.Node, message string) Event {
```

Comment:
Can we model these inflight checks as separate events within the inflightchecks package?

Reply:
I like having a single event to cover all of the inflight checks.  I anticipate more of these as we try to make Karpenter friendlier to use and don't see a need for adding a dozen new event types.